tf-modules/archive/ingest-reporting.tf

```diff
@@ -315,7 +315,7 @@ resource "aws_lambda_function" "publish_pdrs" {
 
 resource "aws_cloudwatch_log_group" "publish_pdrs_logs" {
   name              = "/aws/lambda/${aws_lambda_function.publish_pdrs.function_name}"
-  retention_in_days = 30
+  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "publish_pdr_log_retention", 30)
```

Comment:
To be consistent, this should lookup "archive_publish_pdrs_log_retention" (prefix of "archive_" and changing "pdr" to "pdrs").